Microsoft.Toolkit.Uwp.UI.Controls/WrapPanel/StretchChild.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace Microsoft.Toolkit.Uwp.UI.Controls
+{
+    /// <summary>
+    /// Options for how to calculate the layout of <see cref="Windows.UI.Xaml.Controls.WrapGrid"/> items.
+    /// </summary>
+    public enum StretchChild
+    {
+        /// <summary>
+        /// Don't apply any additional stretching logic
+        /// </summary>
+        None,
+
+        /// <summary>
+        /// Make the first child stretch to push any other items to the end
+        /// </summary>
+        First,
```

Comment:
Reminder that since we don't implement this yet, we should comment it out as a TODO or remove it. 😉

Reply:
Ah, I thought you just didn't want it implemented in the WrapPanel. I'll comment it out